configs/example/gem5_library/riscv-rvv-example.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 Barcelona Supercomputing Center
```

Comment:
Can you copy the full BSD block like the other files. There's some argument that it's not necessary, but we don't want to start that argument again ;). To make everyone happy, please copy the whole block.